cmd/tetragon/main.go

```diff
@@ -713,7 +713,9 @@ func startExporter(ctx context.Context, server *server.Server) error {
 		}()
 	}
 
-	encoder := encoder.NewProtojsonEncoder(writer)
+	// Track how many bytes are written to the event export location
+	encoderWriter := exporter.NewExportdBytesTotalWriter(writer)
```

Comment:
I would have rather handled this in the exporter, but without the ability to wrap the writer before it goes into the encoder, I can't. I didn't want to do a big refactor either, so this is probably the simplest approach.